ee-9/feature-pack/src/main/resources/modules/system/layers/base/jakarta/servlet/jstl/api/main/module.xml

```diff
@@ -25,21 +25,17 @@
 <module xmlns="urn:jboss:module:1.9" name="jakarta.servlet.jstl.api">
 
     <resources>
-        <artifact name="${org.apache.taglibs:taglibs-standard-spec}"/>
-        <artifact name="${org.apache.taglibs:taglibs-standard-impl}"/>
-        <artifact name="${org.apache.taglibs:taglibs-standard-compat}"/>
+        <artifact name="${jakarta.servlet.jsp.jstl:jakarta.servlet.jsp.jstl-api}"/>
+        <artifact name="${org.glassfish.web:jakarta.servlet.jsp.jstl}"/>
     </resources>
 
     <dependencies>
         <module name="java.desktop"/>
         <module name="java.logging"/>
         <module name="java.naming"/>
         <module name="java.sql"/>
-        <!-- org.xml.sax -->
-        <!--WFLY-14219 Remove deprecated <module name="javax.api"/> -->
-        <module name="javax.servlet.api"/>
-        <module name="javax.servlet.jsp.api"/>
-        <module name="org.apache.xalan"/>
+        <module name="jakarta.servlet.api"/>
+        <module name="jakarta.servlet.jsp.api"/>
```

Comment:
For now please use the javax.* alias modules for these 2 deps. I'm trying to keep the use of the javax aliases consistent until something happens that forces a change or we decide to convert everything to jakarta.

Reply:
Fixed @bstansberry .